nss-evening-cohort-14/michelin-star-restaurant-dashboard-michelin-star-restaurant-dashboard-1
# CREATE milestone complete

## Description
Completed the CREATE milestone. When authenticated and logged in, user can create items for staff, reservations, ingredients and menu. When creating a new menu item, the user can select as many ingredients as they would like from a dropdown to be added to the new menu item. The items created appear in Firebase and the DOM is dynamically rendered after each item is created.

## Link to deployed app
https://development--michelin-star-restaurant.netlify.app/#

## Related Issue
#34 
#33 
#32 
#22 

## Motivation and Context
Required for MVP.

## How Can This Be Tested?
visit the deployed link, login, and click on the add button on each page. A  form will appear to submit new values for each item.

## Screenshots (if appropriate):

## Types of changes
